### PR TITLE
Activate Lightning Shield Aura on sim reset

### PR DESCRIPTION
**Pull request by Horatio27**
Activates Lightning Shield Aura on sim reset, effects Enh Shaman.

### CODE REVIEW

File: sim/shaman/lightning_shield.go [hunk 1]
```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/wowsims/wotlk/sim/core/proto"
 )
 
+const defaultLightningSheildStacks = 3
+const staticShockStackMultiplier = 2
```
Comment (johnolos) [inline sim/shaman/lightning_shield.go:12]: Looked into this. Seems it's following convention by "private non-exported constant values" to have `lowerCase`.

**Review by johnolos — APPROVED** [review]
The changes looks good! 👍  from me

File: sim/shaman/lightning_shield.go (hunk 1 above)
Comment (kylebulger27) [reply sim/shaman/lightning_shield.go:12]: just trying to give some names to some magic numbers.

File: sim/shaman/lightning_shield.go [hunk 2]
```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/wowsims/wotlk/sim/core/proto"
 )
 
+const defaultLightningSheildStacks = 3
```
Comment (Wumblesnarf) [inline sim/shaman/lightning_shield.go:11]: Typo, should be "Shield", not "Sheild"

Comment (kylebulger27) [reply sim/shaman/lightning_shield.go:11]: the Sheild always gets me...

File: sim/shaman/lightning_shield.go [hunk 3]
```diff
@@ -66,10 +69,14 @@ func (shaman *Shaman) registerLightningShieldSpell() *core.Spell {
...
+			shaman.activateLightningShieldAura(sim)
 		},
 	})
 
 	return (shaman.LightningShield)
 }
+
+func (shaman *Shaman) activateLightningShieldAura(sim *core.Simulation) {
+	shaman.LightningShieldAura.Activate(sim)
+	shaman.LightningShieldAura.SetStacks(sim, defaultLightningSheildStacks+(staticShockStackMultiplier*shaman.Talents.StaticShock))
```
Comment (Wumblesnarf) [inline sim/shaman/lightning_shield.go:81]: same typo here, also im not sure that replacing "2" with "staticShockStackMultiplier" helps clarity very much, also inconsistent with most other spells

**Pull request merged**